site/content/3.11/_index.md

```diff
@@ -5,6 +5,17 @@ weight: 0
 layout: default
 ---
 
+Normal button
+
+{{< button text="Sign Up" color="green" position="right" >}}
```

Comment:
`text` seems unnecessary/unused as we pass the label text in the body of the shortcode (`.Inner`).

Would does `position` actually do?